src/Component/ContactUsForm.js

```diff
@@ -0,0 +1,95 @@
+import React, { useState} from 'react'
+import '../Style/index.css'
+import axios from 'axios'
+
+// to reset a form
+const INITIAL_STATE = {
+    name:'',
+    email:'',
+    subject:'',
+    message:'',
+}
+
+const ContactUsForm = () => {
+    const [form, setForm] = React.useState({INITIAL_STATE}) 
+    const [formErrors, setformErrors] = useState({})
+   // const [isSubmit, setIsSubmit] = useState(false)
+    const [loading, setLoading] = useState(false);
+
+    const handleChange = (event) => {
+        setForm({...form,
+        [event.target.id]: event.target.value,
+    });
+    }
+    
+    const handleSubmit = (event) => {
+        event.preventDefault();
+        setForm(INITIAL_STATE);
+        setformErrors(validate(form))
+        setLoading(true)
+        axios.post('https://my-json-server.typicode.com/tundeojediran/contacts-api-server/inquiries', {form})
+        .then(response => console.log(response))
+        .catch(err => console.log (err))
+    };
+
+    const validate = (values) => {
+        const errors = {};
+        const regex = /^[^\s@]+@[^\s@]+\.[^\s@]{2,}$/i;
+        if(!values.name) {
+            errors.name = "Name is required" ;
+        }
+        if(!values.email) {
+            errors.email = "Email is required" ;
+        } else if (!regex.test(values.email)) {
+            errors.email = "This is not a valid email format!";
+        }
+        if(!values.message) {
+            errors.message = "Message is required" ;
+        }
+        return errors;
+    }
+    
+     return (
+        <div className="body">
+             {loading && <div>Submitting ...</div>}
+            {Object.keys(formErrors).length === 0 && loading? (<div className='output'> CONGRATULATION! Your submission was successful</div>)
+            : (<div className='output' id='output1'>TRY AGAIN! Your submission was not successful </div>)}
```

Comment:
There is something wrong with this logic you have here. That's why the error message shows at the top of your form